src/main/docker/Dockerfile

```diff
@@ -14,41 +14,54 @@
 # limitations under the License.
 #
 
+# Reference the official Elasticsearch 6 image
+FROM docker.elastic.co/elasticsearch/elasticsearch:6.3.2 AS es6-image
+
 #
-# Preliminary image that downloads and extracts Elasticsearch
+# The actual image definition
 #
-FROM cafapi/opensuse-jre8:1 AS builder
+FROM cafapi/opensuse-jre8:1
 
-RUN zypper -n install which hostname
-RUN zypper -n install unzip
+ENV ELASTIC_CONTAINER=true
```

Comment:
Do we need to add this environment variable?  What is its purpose?

Reply:
This is the only place I found it in the code https://github.com/elastic/elasticsearch/commit/f275a3f07ba0cba170f17b97ffce13854242c1b3.
It is supposed to be used when it depends if elastic search is running in container.